coordinator/src/main/scala/filodb.coordinator/NodeClusterActor.scala

```diff
@@ -208,6 +208,7 @@ private[filodb] class NodeClusterActor(settings: FilodbSettings,
   val shardManager = new ShardManager(settings, assignmentStrategy)
   val localRemoteAddr = RemoteAddressExtension(context.system).address
   var everybodyLeftSender: Option[ActorRef] = None
+  var lastExitRequestMs: Long = System.currentTimeMillis()  // time of previous NodeProtocol.ExitRequest
```

Comment:
Initialized to System.currentTimeMillis() to prevent the case where the ClusterActor repeatedly kills its own JVM.

TODO: add comment about this

Reply:
No longer applicable